core/wiki/macros/list-links-horizontal-draggable.tid

```diff
@@ -0,0 +1,68 @@
+title: $:/core/macros/list-links-horizontal-draggable
+tags: $:/tags/Macro $:/tags/Global
+
+\whitespace trim
+\procedure list-links-horizontal-draggable-caption()
+<$let tv-wikilinks="no">
+	<span class="tc-small-gap-right">
+		<$transclude field="caption">
+			<$view field="title"/>
+		</$transclude>
+	</span>
+</$let>
+\end
+
+\procedure list-links-horizontal-draggable(tiddler,field:"list",emptyMessage,type:"span",subtype:"span",class:"",itemTemplate,dragHandle:"⠿",enable:"yes")
+	\procedure tagSort() [<tiddler>get<field>enlist-input[]] :sort:alphanumeric:caseinsensitive[subfilter{$:/config/Tags/CustomSort/subfilter}]
+	\procedure defaultSort() [<tiddler>get<field>enlist-input[]]
+	\function tf.customSort() [<field>match[tags]then<tagSort>else<defaultSort>]
+
+<span class="tc-horizontal-draggable tc-tiny-gap-right">
+	<$let targetTiddler=<<tiddler>>
+		targetField=<<field>>
+		_enable={{{ [<tv-enable-drag-and-drop>!match[no]then<enable>else[yes]] }}}
+		_dragHandle={{{ [<_enable>match[yes]then<dragHandle>else[]] }}}
+	>
+		<$genesis $type=<<type>> class=<<class>>>
+			<$list filter=<<tf.customSort>> emptyMessage=<<emptyMessage>>>
+				<$droppable
+					actions=<<list-links-draggable-drop-actions>>
+					tag=<<subtype>>
+					enable=<<_enable>>
+				>
+					<span class="tc-horizontal-draggable-placeholder"/>
+					<$draggable tiddler={{!!title}} tag="span"
+						enable=<<_enable>>
+						class="tc-horizontal-draggable-item"
+					>
+						<% if [<_dragHandle>!is[blank]] %>
+							<span class="tc-draggable-handle tc-tiny-gap-right">
+								<<_dragHandle>>
+							</span>
+						<% endif %>
+						<% if [<itemTemplate>is[variable]] %>
```

Comment:
I am not at all keen on overloading the `itemTemplate` parameter in this way. No existing core macro/procedure works like this, and I think it is guaranteed to lead to unexpected and hard to debug errors. It would be clearer to have two separate parameters, with the precedence between them documented.